src/PowerShellEditorServices/Language/LanguageService.cs

```diff
@@ -530,44 +538,40 @@ public FindOccurrencesResult FindSymbolsInFile(ScriptFile scriptFile)
             {
-                CommandInfo commandInfo =
-                    await CommandHelpers.GetCommandInfo(
-                        foundSymbol.SymbolName,
-                        this.powerShellContext);
+                return null;
+            }
 
-                if (commandInfo != null)
-                {
-                    try
-                    {
-                        IEnumerable<CommandParameterSetInfo> commandParamSets = commandInfo.ParameterSets;
-                        return new ParameterSetSignatures(commandParamSets, foundSymbol);
-                    }
-                    catch (RuntimeException e)
-                    {
-                        // A RuntimeException will be thrown when an invalid attribute is
-                        // on a parameter binding block and then that command/script has
-                        // its signatures resolved by typing it into a script.
-                        this.logger.WriteException("RuntimeException encountered while accessing command parameter sets", e);
+            CommandInfo commandInfo =
+                await CommandHelpers.GetCommandInfo(
+                    foundSymbol.SymbolName,
+                    _powerShellContext);
 
-                        return null;
-                    }
-                    catch (InvalidOperationException)
-                    {
-                        // For some commands there are no paramsets (like applications).  Until
-                        // the valid command types are better understood, catch this exception
-                        // which gets raised when there are no ParameterSets for the command type.
-                        return null;
-                    }
-                }
-                else
-                {
-                    return null;
-                }
+            if (commandInfo == null)
+            {
+                return null;
             }
-            else
+
+            try
             {
+                IEnumerable<CommandParameterSetInfo> commandParamSets = commandInfo.ParameterSets;
+                return new ParameterSetSignatures(commandParamSets, foundSymbol);
+            }
+            catch (RuntimeException e)
+            {
+                // A RuntimeException will be thrown when an invalid attribute is
+                // on a parameter binding block and then that command/script has
+                // its signatures resolved by typing it into a script.
+                _logger.WriteException("RuntimeException encountered while accessing command parameter sets", e);
+
+                return null;
+            }
+            catch (InvalidOperationException)
+            {
+                // For some commands there are no paramsets (like applications).  Until
+                // the valid command types are better understood, catch this exception
+                // which gets raised when there are no ParameterSets for the command type.
```

Comment:
I wonder if we should log this as a diagnostic level message - we expect this exception in some cases but just in case we're getting this for cases we didn't expect, it might be good to have a log entry when log level set to Diagnostic.